test/runtests.jl

```diff
@@ -139,13 +139,22 @@ using Test, SparseArrays, Missings
     @test_throws MethodError disallowmissing([missing missing])
 
     # Lifting
+    struct CubeRooter end
+    (::CubeRooter)(x) = cbrt(x)
+    cuberoot = CubeRooter()
+    @test passmissing(cuberoot)(27) == 3.0
+    @test isequal(passmissing(cuberoot)(missing), missing)
     @test passmissing(sqrt)(4) == 2.0
     @test isequal(passmissing(sqrt)(missing), missing)
     @test isequal(passmissing(sqrt).([missing, 4]), [missing, 2.0])
     @test passmissing((x,y)->"$x $y")(1, 2) == "1 2"
     @test isequal(passmissing((x,y)->"$x $y")(missing), missing)
     @test_throws ErrorException passmissing(string)(missing, base=2)
 
+    @test passmissing(sin) === Missings.PassMissing{typeof(sin)}(sin)
```

Comment:
We also might add tests `passmissing(Int)(1)` and `passmissing(Int)(missing)`

@ararslan Do you think that with `Core.Typeof` the speed of the above will increase?

Reply:
added :)